libsolidity/ast/ASTJsonExporter.cpp

```diff
@@ -214,7 +214,8 @@ bool ASTJsonExporter::visit(SourceUnit const& _node)
 {
 	std::vector<pair<string, Json::Value>> attributes = {
 		make_pair("license", _node.licenseString() ? Json::Value(*_node.licenseString()) : Json::nullValue),
-		make_pair("nodes", toJson(_node.nodes()))
+		make_pair("nodes", toJson(_node.nodes())),
+		make_pair("experimentalSolidity", _node.experimentalSolidity() ? Json::Value(_node.experimentalSolidity()) : Json::nullValue)
 	};
 
```

Comment:
I'd prefer
```suggestion
	};
	if (_node.experimentalSolidity())
		attributes.emplace_back("experimentalSolidity", true)
```
That's also how we usually do this elsewhere - but also fine as is, since it seems to work.

Reply:
I've seen both approaches used, but I can change it, don't really have a preference.